src/components/edit/earthEngine/BandSelect.js

```diff
@@ -5,16 +5,29 @@ import { connect } from 'react-redux';
 import { SelectField } from '../../core';
 import { setBand } from '../../../actions/layerEdit';
 
-const BandSelect = ({ band = [], bands, setBand, errorText }) => (
-    <SelectField
-        label={i18n.t('Groups')}
-        items={bands}
-        multiple={true}
-        value={band}
-        onChange={setBand}
-        errorText={!band.length && errorText ? errorText : null}
-    />
-);
+const MAX_BANDS = 10;
+
+const BandSelect = ({ band = [], bands, setBand, errorText }) => {
+    let warning;
+
+    if (band.length > MAX_BANDS) {
+        warning = i18n.t(
+            'Choosing many groups takes a long time to calculate and display.'
```

Comment:
Because calculations may take a long time, a maximum of ${MAX_BANDS} groups is allowed

Reply:
We have changed the logic so MAX_BANDS is no longer a restriction, but we show a warning to the user. I renamed the MAX_BANDS to WARNING_BANDS and added a code comment to make it clearer.